src/ls/Lsp_util.mli

```diff
@@ -0,0 +1,2 @@
+val range_of_location : Semgrep_output_v1_t.location -> Lsp.Types.Range.t
```

Comment:
Those are actually not utilities related to LSP but utilities related to Semgrep.
Maybe Convert_utils.mli